src/main/java/seedu/momentum/model/project/UniqueProjectList.java

```diff
@@ -97,6 +104,69 @@ public void setProjects(List<Project> projects) {
         internalList.setAll(projects);
     }
 
+    /**
+     * Sets the order of the list of projects according to given {@code sortType} and {@code isAscending}.
+     *
+     * @param sortType type of sort.
+     * @param isAscending order of sort.
+     */
+    public void setOrder(SortType sortType, boolean isAscending) {
+
+        requireNonNull(sortType);
+
+        Comparator<Project> nameCompare = new NameCompare();
+
+        switch (sortType) {
+        case ALPHA:
+            nameCompare = isAscending ? nameCompare : nameCompare.reversed();
+            this.sortType = SortType.ALPHA;
+            internalList.sort(nameCompare);
```

Comment:
The method is quite long, maybe extract this into another method?

Reply:
Fixed!